pkg/arvo/lib/btc-node-json.hoon

```diff
@@ -0,0 +1,1080 @@
+/-  *btc-node-hook
+=,  format
+|%
+++  btc-rpc
+  =,  ^btc-rpc
+  ::  Utility core
+  ::
+  =+  |%
+      ::  Ideally +fall would have worked here, but we need to supply
+      ::  the type of json (%s, %b...) only when unit is non empty so
+      ::  this is like a reverse +bond (applied to the non-null case)
+      ::  or a +bif that doesn't return a +unit
+      ::
+      ++  ferm  |*([a=(unit) t=term] ?~(a ~ t^u.a))
+      ::
+      ::  We want to make sure that numbers are @ud but JSON
+      ::  defines numbers as @ta, so we need to do a re-parse
+      ::
+      ++  feud  |*(a=(unit) ?~(a ~ n+(scot %ud u.a)))
+      ::
+      ::  %method:
+      ::  Removes 'hep' (-) from a %tas producing the RPC
+      ::   method to use in a call
+      ::
+      ::  (e.g. %add-multisig-address -> 'addmultisigaddress')
+      ::
+      ++  method
+        |=  t=@t
+        ^-  @t
+        %+  scan  (scow %tas t)
+          ((slug |=(a=[@ @t] (cat 3 a))) (star hep) alp)
+      ::
+      ++  bip125
+        ::  This would have been more elegant:
+        ::  ['bip125-replaceable' (cu term so)]
+        ::  but a term is not of type ?(%yes %no %unknown)
+        ::  so a manual convertion seems the only(?) way.
+        ::
+        |=  t=@t
+        ^-  bip125-replaceable
+        ?:  =(t 'yes')
+          %yes
+        ?:  =(t 'no')
+          %no
+        ?:  =(t 'unknown')  !!
```

Comment:
This should be `?.` but will probably removed in favor of using the mold directly